spec/models/user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'hello man') }
+
+  before { subject.save }
+
+  it 'Posts should be not be Valid' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+  it 'User post counter to increment' do
+    subject.author = User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                              bio: 'Teacher from Mexico.')
+    subject.send(:posts_counter)
+    expect(subject.author.posts_counter).to be(1)
```

Comment:
- [ ] According to the project requirement, 👇🏻 Any custom methods in the models should have a corresponding unit test. You're missing to add unit test of the  `last_three_posts` method of your user model, ` comments_counter` method of your comment model.... please address this ✍🏻 

![im2](https://user-images.githubusercontent.com/79058364/215210008-f647c979-9f49-40cc-a73e-e5d3c29cfbb1.png)

Reply:
Thanks for the reminder. Issue is adressed.  😄